recipe/patches/0005-Add-missing-any_invocable-target-to-absel-bazel.patchFrom 7a73e8691bd7f6c66eb887520576482e9cee9f87 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Sun, 2 Jul 2023 05:36:41 +0000
Subject: [PATCH 05/10] Add missing any_invocable target to absel bazel

---
 third_party/absl/system.absl.functional.BUILD | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/third_party/absl/system.absl.functional.BUILD b/third_party/absl/system.absl.functional.BUILD
index a4f70acf..f05aa98d 100644
--- a/third_party/absl/system.absl.functional.BUILD
+++ b/third_party/absl/system.absl.functional.BUILD
@@ -9,3 +9,7 @@ cc_library(
 cc_library(
     name = "function_ref",
 )
+
+cc_library(
+    name = "any_invocable",
+)
-- 
2.39.2 (Apple Git-143)

